site/visitor-console/src/components/Registration/Registration.tsx

```diff
@@ -19,6 +20,9 @@ const Registration = (props: Props) => {
         label: string;
     }
     const genderOptions: selectOption[] = ["Male", "Female", "Other"].map(d => {return {value: d, label: d}});
+    const gradSemesterOptions: selectOption[] = ["Fall", "Spring", "Summer"].map(d => {return {value: d, label: d}});
+    const gradYearOptions: selectOption[] = ["2022", "2023", "2024", "2025", "2026", "2027",
```

Comment:
Awesome! Yeah I was looking into making it dynamically generate, but I thought it would be a lot messier than this, I went ahead and made the changes to enable this.
I meant for the graduation changes to be in its own pull request and not mixed with the PITR, so I removed the changes from this merge, and have opened a request at #64 to handle the grad date changes.